airflow/www/templates/airflow/dags.html

```diff
@@ -103,13 +103,11 @@ <h2>DAGs</h2>
 
                 <!-- Column 7: Last Run -->
                 <td class="text-nowrap latest_dag_run {{ dag.dag_id }}">
-                  {% set last_run = dag.get_last_dagrun(include_externally_triggered=True) %}
-                  {% if last_run and last_run.execution_date %}
-                    <a href="{{ url_for('Airflow.graph', dag_id=dag.dag_id, execution_date=last_run.execution_date) }}">
-                      {{ last_run.execution_date.strftime("%Y-%m-%d %H:%M") }}
-                    </a>
-                    <span aria-hidden="true" id="statuses_info" title="Start Date: {{ last_run.start_date.strftime("%Y-%m-%d %H:%M") }}" class="glyphicon glyphicon-info-sign"></span>
-                  {% endif %}
+                  <div height="10" width="10" id='last-run-{{ dag.safe_dag_id }}' style="display: block;">
+                    <a></a>
+                    <img class="loading-last-run" width="15" src="{{ url_for("static", filename="loading.gif") }}">
```

Comment:
This image is wider than the div it's in - does it look okay?

Reply:
It looked ok to me, I agree probably the settings can be cleaned up and this width is strange. This is copied from all of the other async loaded components.